packages/babel-preset-env-standalone/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "babel-preset-env-standalone",
```

Comment:
This name is a bit long. What do you think of calling it `babel-standalone-env` instead?